crates/re_ui/examples/re_ui_example.rs

```diff
@@ -600,3 +645,788 @@ impl egui_tiles::Behavior<Tab> for MyTileTreeBehavior {
+                    Some(DropTarget::new(
+                        (response.rect.left() + indent..=response.rect.right()).into(),
+                        top.top(),
+                        previous_container_id,
+                        usize::MAX,
+                    ))
+                } else {
+                    Some(DropTarget::new(
+                        response.rect.x_range(),
+                        top.top(),
+                        parent_id,
+                        pos_in_parent,
+                    ))
+                }
+            } else if !is_container {
+                if ui.rect_contains_pointer(bottom) {
+                    // insert after me
+                    Some(DropTarget::new(
+                        response.rect.x_range(),
+                        bottom.bottom(),
+                        parent_id,
+                        pos_in_parent + 1,
+                    ))
+                } else {
+                    None
+                }
+            } else {
+                let body_rect = body_response.map(|r| r.rect).filter(|r| r.width() > 0.0);
+                if let Some(body_rect) = body_rect {
+                    if ui.rect_contains_pointer(left_bottom) || ui.rect_contains_pointer(bottom) {
+                        // insert at pos = 0 inside me
+                        Some(DropTarget::new(
+                            (body_rect.left() + indent..=body_rect.right()).into(),
+                            left_bottom.bottom(),
+                            item_id,
+                            0,
+                        ))
+                    } else if ui.rect_contains_pointer(content_left_bottom) {
+                        // insert after me in my parent
+                        Some(DropTarget::new(
+                            response.rect.x_range(),
+                            content_left_bottom.bottom(),
+                            parent_id,
+                            pos_in_parent + 1,
+                        ))
+                    } else {
+                        None
+                    }
+                } else if ui.rect_contains_pointer(left_bottom) {
+                    // insert after me in my parent
+                    Some(DropTarget::new(
+                        response.rect.x_range(),
+                        left_bottom.bottom(),
+                        parent_id,
+                        pos_in_parent + 1,
+                    ))
+                } else if ui.rect_contains_pointer(bottom) {
+                    // insert at pos = 0 inside me
+                    Some(DropTarget::new(
+                        (response.rect.left() + indent..=response.rect.right()).into(),
+                        bottom.bottom(),
+                        item_id,
+                        0,
+                    ))
+                } else {
+                    None
+                }
+            }
+        }
+    }
+
+    struct DropTarget {
```

Comment:
This should have a docstring explaining it ("this is a drop target within a container, visualized as a horizontal line, …")